README.md

```diff
@@ -211,9 +211,16 @@ First, start postgres with:
 $ npm run postgres
 ```
 
-Then in another terminal:
+Then you can, in another terminal, find the running docker, init the DB, then run the tests:
 
 ```
+$ docker ps
```

Comment:
I dont think we should make the example docker specific so just specifying below should be ok

Then, in another terminal, you can init the test database and run the tests:

```
$ psql -d postgres -U postgres  -c 'CREATE TABLE users(id serial PRIMARY KEY, username VARCHAR (50) NOT NULL);'
$ npm test
```

Reply:
@temsa sorry ignore this